Tests/NIOPosixTests/BootstrapTest.swift

```diff
@@ -448,7 +448,7 @@ class BootstrapTest: XCTestCase {
         }
 
         // But now, it should be there.
-        XCTAssertNoThrow(_ = try server.pipeline.context(name: "AcceptHandler").wait())
+        XCTAssertNoThrow(_ = try server.pipeline.context(name: "AcceptHandler").map { _ in () }.wait())
```

Comment:
Why is this `map` needed?

Reply:
Because the handler is not `Sendable` so we get a warning when we `wait` for it. So mapping to Void avoids that :)